journal-of-historical-linguistics.csl

```diff
@@ -111,15 +111,13 @@
           <text macro="title" prefix=" "/>
           <group prefix=".">
             <group>
-              <group suffix=".">
-                <names variable="editor translator">
-                  <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always" prefix=" In: " suffix=","/>
+              <group prefix=" " suffix=".">
+                <text term="in" text-case="capitalize-first" suffix=" "/>
+                <names variable="editor translator" delimiter="," suffix=",">
+                  <name delimiter=" " and="symbol" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
That same example also suggests that there shouldn't be an "in" and that the label should be before the name and be `verb-short`.